packages/css/src/components/text-area/README.md

```diff
@@ -0,0 +1,21 @@
+# TextArea
+
+A form field that allows the user to input text over multiple lines.
+
+## Guidelines
+
+- Use a TextArea when users need to enter more than 1 sentence of text, such as a comment or description.
+- The height of the TextArea should be appropriate for the information to be entered.
+  - Use cols and rows attributes to set the width and height of the TextArea.
+  - Use the experimental grow attribute to allow the TextArea to grow in height as the user types, this only works in modern browsers.
```

Comment:
The current implementation breaks the layout for browsers that do not support this property, again because of the `width: 100%` thing. We should decide what we want to do there

Reply:
Width is set to auto for textareas that grow. Browser support is > 97%.

Reply:
https://caniuse.com/?search=form-sizing%3A%20content

Reply:
You're not using `form-sizing`, but `field-sizing`.

Reply:
Don't remember how I got that reference but I still think we should leave it in :)

Reply:
Well, the current implementation reverts TextArea to its native sizing for browsers that do not support `field-sizing` (which is pretty much all of them, I think), which is kind of unusable. So in its current implementation, I would not leave it in.

Reply:
removed